### ISSUE TITLE
Import bug for DOI 	10.1007/s00222-021-01079-5

### ISSUE THREAD

**Issue by rsdoiel** [caltechlibrary/acacia#19]
The DOI 

       10.1007/s00222-021-01079-5

Did not import into EPrints for Tony. The error messages didn't say why.  Determine the problem, solve or give a better error message.

**Comment by rsdoiel** [comment]
George ran into this same error message when importing a list. 

```
I sent the following 21 DOIs to ACACIA

I received a processing error for 10.3847/1538-4365/ac35e9, although the DOI is fully registered and functional.

Trying to import records is consistently resulting in 

Sorry, Acacia encountered an error
400, Bad Request

I've tried them locally with the eprint tool.  The same DOI chokes, but the other 20 records are successfully created.

George

﻿On 2/3/22, 11:42 AM, "George Porter" <[george@library.caltech.edu](mailto:george@library.caltech.edu)> wrote:

   https://doi.org/10.3847/1538-4357/ac3aee
   https://doi.org/10.3847/1538-4357/ac403a
   10.3847/1538-3881/ac4334
   https://doi.org/10.3847/PSJ/ac4580
   https://doi.org/10.3847/1538-4357/ac36d2
   https://doi.org/10.3847/1538-4357/ac332e
   https://doi.org/10.3847/1538-3881/ac4334
   https://doi.org/10.3847/1538-4357/ac38a5
   https://doi.org/10.3847/1538-4357/ac36c8
   https://doi.org/10.3847/1538-4357/ac3766
   https://doi.org/10.3847/1538-4357/ac3ede
   https://doi.org/10.1088/1538-3873/ac46ba
   https://doi.org/10.3847/PSJ/ac4493
   https://doi.org/10.3847/1538-4357/ac313f
   https://doi.org/10.3847/PSJ/ac4581
   https://doi.org/10.3847/1538-4357/ac36c8
   https://doi.org/10.3847/PSJ/ac457f
   https://doi.org/10.3847/1538-4357/ac3829
   https://doi.org/10.3847/1538-4357/ac3b4a
   https://doi.org/10.3847/1538-4365/ac35e9
   https://doi.org/10.3847/1538-4357/ac38a3
```

**Comment by tmorrell** [comment]
This DOI is not in the CrossRef metadata yet. It's a bit weird because the DOI does resolve, but the metadata probably hasn't propagated yet. If the metadata is blank Acacia should put up a warning and let the user know to try again later. I'm fairly certain this issue will resolve itself in the next day or so.

**Comment by rsdoiel** [comment]
This turns out to be a bug in ep3apid 1.1.2 and is fixed in 1.1.3-next. Aside from wrong mime type of POST to the API from Acacia the key bug was in ep3apid.
